ts/src/main/scala/ch/epfl/ts/traders/SimpleFXTrader.scala

```diff
@@ -1,40 +1,47 @@
 package ch.epfl.ts.traders
 
-import ch.epfl.ts.component.Component 
+import ch.epfl.ts.component.Component
 import ch.epfl.ts.indicators.MA
 import ch.epfl.ts.data.Currency._
-import ch.epfl.ts.data.{MarketAskOrder, MarketBidOrder, Quote}
+import ch.epfl.ts.data.{ MarketAskOrder, MarketBidOrder, Quote }
+import ch.epfl.ts.data.Currency
 
 /* This simple trader will use two moving average and send order when this two MA cross each other.
  * @ Param the length of the two moving average period.
  */
-class SimpleFXTrader(val uid: Long, val shortPeriod: Int, val longPeriod : Int, val volume : Double) extends Component{
-  
+
+//symbol format  EUR/CHF , CHF/USD .. , as string => easier for user. 
+class SimpleFXTrader(val uid: Long, val symbol: String, val shortPeriod: Int, val longPeriod: Int, val volume: Double) extends Component {
```

Comment:
Same comment here about the symbol, we should use our `Currency` type to catch potential user errors early on (i.e. compile time rather than runtime).